agreement/player_test.go

```diff
@@ -37,7 +37,7 @@ func init() {
 }
 
 func makeTimeoutEvent() timeoutEvent {
-	return timeoutEvent{T: timeout, RandomEntropy: crypto.RandUint64()}
+	return timeoutEvent{T: timeout, RandomEntropy: crypto.RandUint64(), Proto: ConsensusVersionView{Version: protocol.ConsensusCurrentVersion}}
```

Comment:
It seem like this change means all tests with the dynamic filter turned on, where they used to run with it turned off. (Actually, I don't know how `Proto` was set before.)